src/main/java/frc/robot/Constants.java

```diff
@@ -1,36 +1,47 @@
 package frc.robot;
 
-public class Constants {
-  // PCM
-  public static final int kPCMId = 0;
-  public static final int kIntakeSolenoidForwardId = 2;
-
-  // DIO
-  public static final int kExtensionUpperLimitPort = 2;
-  public static final int kExtensionLowerLimitPort = 1;
-  public static final int kPivotLowerLimitPort = 0;
-
-  // Intake
-  public static final int kIntakeMasterId = 14;
-
-  // Elevator
-  public static final int kElevatorPivotMotorId = 10;
-  public static final int kElevatorExtensionMotorId = 9;
-
-  // Drivetrain
-  public static final int kDrivetrainFLMotorId = 5;
-  public static final int kDrivetrainBLMotorId = 6;
-  public static final int kDrivetrainFRMotorId = 7;
-  public static final int kDrivetrainBRMotorId = 8;
-
-  // Pivot set points
-  public static final double kPivotGroundCount = 0;
-  public static final double kPivotScoreCount = -55;
-  public static final double kPivotPreScoreCount = -65;
-  public static final double kPivotStowCount = -120;
-
-  // Extension set points
-  public static final double kExtensionStowCount = 0;
-  public static final double kExtensionMidGoalCount = 34;
-  public static final double kExtensionHighGoalCount = 72;
+public final class Constants {
+  public class Drivetrain {
+    // Drivetrain wheel offsets
+    // TODO: Change for final robot
+    public static final double kXDistance = 0.762; // 30 inches
+    public static final double kYDistance = 0.762; // in meters!
+
+    public static final double kXCenterDistance = kXDistance / 2.0;
+    public static final double kYCenterDistance = kYDistance / 2.0;
+
+    // Drivetrain drive motor constants
+    public class Drive {
+      public static final int kFLDriveMotorId = 5;
```

Comment:
```suggestion
      public static final int kFLMotorId = 5;
```

Remove redundant naming when inside of a scope with that name